.github/workflows/ci.yml

```diff
@@ -100,6 +100,25 @@ jobs:
       - name: "Run tests"
         run: cargo test --manifest-path=compiler/Cargo.toml --locked ${{ matrix.target.features && '--features' }} ${{ matrix.target.features }}
 
+  
+  build-docs:
+    name: Build docs (${{ matrix.target.os }})
```

Comment:
This name could be confusing since it kinda sounds like this is building our documentation website. Maybe instead call it "Build Cargo Docs" or something?

For bonus points, maybe we could figure out how to build this as part of the actual website build and include it as a part of the website!